custom-docker/build.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF 1>&2
+Usage: $0 [-h] [-a <ARCHITECTURES>] [-r <VERSION>]
+Builds the Trino Docker image
+
+-h       Display help
+-a       Build the specified comma-separated architectures, defaults to amd64,arm64,ppc64le
+-t       Image tag name, defaults to trino-custom
+-r       Build the specified Trino release version, downloads all required artifacts
+-x       Skip image tests
+EOF
+}
+
+ARCHITECTURES=(amd64 arm64 ppc64le)
+# also make sure that Dockerfile uses the same version
+TRINO_VERSION='472'
+echo "Trino version set to ${TRINO_VERSION}"
+TAG_PREFIX=trino-custom
+
+SKIP_TESTS=true
```

Comment:
Unused variable, consider running the `shellcheck` linter on the script to fix all issues like this.